headers/types/common/sound.h

```diff
@@ -0,0 +1,671 @@
+// Types used by the DSE sound library.
+
+#ifndef HEADERS_TYPES_SOUND_H_
```

Comment:
The define needs updating

Reply:
I was looking at `window.h`, but I see that this one is also wrong. I'll fixed both.

Reply:
Oops, thanks.